.travis.yml

```diff
@@ -3,7 +3,7 @@ scala:
    - 2.13.0
 
 jdk:
- - oraclejdk8
+ - openjdk8
```

Comment:
cheeky

Reply:
Build broke lol, hopefully my change fixes it.

I'll monitor.